halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesCommandProperties.java

```diff
@@ -69,4 +69,8 @@
   static final String CACHE_THREADS =
       "Number of caching agents for this kubernetes account. Each agent handles a subset of the namespaces available to this account. "
           + "By default, only 1 agent caches all kinds for all namespaces in the account.";
+  static final String CUSTOM_RESOURCES =
+      "(V2 Only) List of Kubernetes Custom Resources to be cached by clouddriver and available for use in patch and delete pipeline stages. "
+          + "Fields besides the Kubernetes Kind can be set using the following "
```

Comment:
I think these next two lines need to be updated now that we have `--spinnaker-kind` and `--versioned` flags